src/Link.js

```diff
@@ -4,8 +4,19 @@ export function Link(props, children) {
   props.href = props.to
   props.to = null
 
-  props.onclick = function(element) {
-    element.preventDefault()
+  props.onclick = function(event) {
+    if (
+      event.button !== 0 ||
```

Comment:
Agreed 👍 

If people have a compelling use case for it, then it's not a very tough addition down the road.